crates/uv/tests/pip_sync.rs

```diff
@@ -65,14 +65,32 @@ fn missing_venv() -> Result<()> {
     ----- stdout -----
 
     ----- stderr -----
-    error: No virtual environment found
+    error: No virtual environment found; run `uv venv` to create an environment, or pass `--system` to install into a non-virtual environment
     "###);
 
     assert!(predicates::path::missing().eval(&context.venv));
 
     Ok(())
 }
 
+#[test]
+fn missing_system() -> Result<()> {
+    let context = TestContext::new_with_versions(&[]);
+    let requirements = context.temp_dir.child("requirements.txt");
+    requirements.write_str("anyio")?;
+
+    uv_snapshot!(context.filters(), context.pip_sync().arg("requirements.txt").arg("--system"), @r###"
+    success: false
+    exit_code: 2
+    ----- stdout -----
+
+    ----- stderr -----
+    error: No system environment found; run `uv python install` to install Python
```

Comment:
So there's a problem here, in that if you install with `uv python install` we actually use an externally managed marker and won't let you install into it with `--system` :)

Reply:
Should I just remove this part then?

Reply:
Maybe yeah. Is this error only surfaced in `uv pip`? If it's surfaced in the top-level commands this _would_ be relevant. I feel like we say something else there though? You might want to say "No system Python installation found" instead of this since people don't really know what a "system environment" is?

Reply:
Done. It's only in `uv pip` yeah.